pkg/resource/cluster/hooks.go

```diff
@@ -262,3 +263,123 @@ func (rm *resourceManager) newMemoryDBClusterUploadPayload(
 
 	return res
 }
+
+// getTags gets tags from given ParameterGroup.
+func (rm *resourceManager) getTags(
+	ctx context.Context,
+	resourceARN string,
+) ([]*svcapitypes.Tag, error) {
+	resp, err := rm.sdkapi.ListTagsWithContext(
+		ctx,
+		&svcsdk.ListTagsInput{
+			ResourceArn: &resourceARN,
+		},
+	)
+	rm.metrics.RecordAPICall("GET", "ListTags", err)
+	if err != nil {
+		return nil, err
+	}
+	tags := make([]*svcapitypes.Tag, 0, len(resp.TagList))
+	for _, tag := range resp.TagList {
```

Comment:
You've implemented `sdkTagsFromResourceTags` below; consider adding a function `resourceTagsFromSDKTags` to do this, which is the inverse transformation.

Reply:
Done